system/Makefile.am

```diff
@@ -3,6 +3,7 @@
 #
 MAINTAINERCLEANFILES= $(srcdir)/Makefile.in
 CLEANFILES = \
+	netdata.logrotate \
```

Comment:
I am sooooooooo sorry... but this is not sorted... should be before the netdata.service, if you use vi, just select the lines and :sort